analyzer/optane/check_dimm_MediaTemperatureInjectionEnabled

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+#set -x
+
+# Prevent repeated sourcing
+if [ -n "${__MODULE_CHECK_DIMM_MediaTemperatureInjectionEnabled_}" ]; then
+  return
+fi
+__MODULE_CHECK_DIMM_MediaTemperatureInjectionEnabled_="Loaded"
+
+# Validate DIMM PackageSparingCapable != 1 (True)
+function optane_check_MediaTemperatureInjectionEnabled() {
+  local FNAME=$1        # File name to process
+  local ERR_STATE=false # Used for error reporting
+
+  # Exit the function with INFO message if the file to process doesn't exist
+  if [[ ! -f ${FNAME} ]] ; then
+    info_msg " Could not process '${FNAME}'. File not found"
+    REPORT_COUNT_INFO=$((REPORT_COUNT_INFO+1)) # Increment the INFO count
+    return
+  fi
+
+  # Create Associative Array
+  declare -A MediaTemperatureInjectionEnabled  # Key is the DIMM Property value; Value is a list of DIMMs with this Property value
+
+  # Get the DimmID, Property, and Value fields from the input file
+  while IFS='|' read -r DimmID Property Value 
+  do
+    DimmID="$(echo ${DimmID})"         # Remove leading and trailing whitespaces
+    Property="$(echo ${Property})" # Remove leading and trailing whitespaces
+    PropertyValue="$(echo ${Value})"   # Remove leading and trailing whitespaces
+    if [ ${OPT_VERBOSITY} -ge 3 ]; then
+      debug_msg "DimmID ${DimmID}: ${Property}=${Value}"
+    fi
+    # Report PMem Modules that have PackageSparingCapable  = 0 (False/Disabled) 
+    if [[ "${PropertyValue}" -ne 0 ]]; then
+      ERR_STATE=true
+    fi
+    # Add the DIMM to the associative array 
+   MediaTemperatureInjectionEnabled[${PropertyValue}]+="${DimmID},"
+  done <<<  "$(${GREP} MediaTemperatureInjectionEnabled ${FNAME})" || return 0 
+
+  # Report DIMMS with ManageabilityState = "Unmanageable"
+  if [ "${ERR_STATE}" = true ]; then
+    info_msg "Multiple (${#MediaTemperatureInjectionEnabled[@]}) ${Property} DIMM states found!"
+    for d in "${!MediaTemperatureInjectionEnabled[@]}" ; do # Display the full list of DIMMs found
+      MediaTemperatureInjectionEnabled=${MediaTemperatureInjectionEnabledU[${d}]::-1} # Strip the last character ',' from the string
```

Comment:
Typo with 'MediaTemperatureInjectionEnabledU' should be 'MediaTemperatureInjectionEnabled' (No 'U')